clickhouse/provider.go

```diff
@@ -126,13 +125,15 @@ func (p *clickhouseProvider) Configure(ctx context.Context, req provider.Configu
 	// Default values to environment variables, but override
 	// with Terraform configuration value if set.
 
-	environment := os.Getenv("CLICKHOUSE_ENV")
+	apiUrl := os.Getenv("CLICKHOUSE_API_URL")
 	organizationId := os.Getenv("CLICKHOUSE_ORG_ID")
 	tokenKey := os.Getenv("CLICKHOUSE_TOKEN_KEY")
 	tokenSecret := os.Getenv("CLICKHOUSE_TOKEN_KEY")
 
-	if !config.Environment.IsNull() {
-		environment = config.Environment.ValueString()
+	if !config.ApiUrl.IsNull() {
+		apiUrl = config.ApiUrl.ValueString()
+	} else {
+		apiUrl = "https://api.clickhouse.cloud/v1"
 	}
```

Comment:
If a URL is not provided in the provider config and I want to use env var, it will be overridden with line 136 anyway.